server/src/server-settings.ts

```diff
@@ -29,14 +29,7 @@ function nodeIpAddress(family: number): string[] {
         ...costlyNetworks,
     ];
 
-    let interfaces: any;
```

Comment:
Feel like this change was q mistake. I don't think the server should be responsible for managing quirks like this.